massa-execution/src/interface_impl.rs

```diff
@@ -271,4 +343,23 @@ impl Interface for InterfaceImpl {
         // TODO store the event somewhere
         Ok(())
     }
+
+    /// Returns the current time (millisecond unix timestamp)
+    fn get_time(&self) -> Result<u64> {
+        let context = context_guard!(self);
```

Comment:
It would be better to avoid holding the lock for longer than necessary. For example you could do the below:

let slot = {
    let context = context_guard!(self);
    context.slot
};

Reply:
done